python/pyrogue/pydm/widgets/variable_tree.py

```diff
@@ -182,6 +198,11 @@ def __init__(self,*,path,top,parent,variable):
 
         self._top._tree.setItemWidget(self,3,w)
 
+        width = fm.width('0xAAAAAAAA    ')
```

Comment:
0xAAAAAAAA seems strange. Do you need all that white-space in the python string argument?